packages/common-lib/src/services/commentRegistryService.js

```diff
@@ -0,0 +1,88 @@
+import { get, post, update as coreUpdate } from './RestClient'
+import mapInterfaceData from './mapInterfaceData'
+import manifest from '../manifest.json'
+
+const interfaceData = {
+  id: 'commentId',
+  contextId: 'contextId',
+  context: 'context',
+  userId: 'userId',
+  comment: 'comment',
+  parentId: 'parentId',
+  status: 'status',
+  privacy: 'privacy'
+}
+
+let only = Object.keys(interfaceData)
```

Comment:
variable name looks  incorrect

Reply:
does this looks ok?
`const commentEntityAttributes = Object.keys(interfaceData) `